internal/plugins/helm/v1/scaffolds/init.go

```diff
@@ -79,15 +79,23 @@ func (s *initScaffolder) scaffold() error {
 	if err := os.MkdirAll(chartutil.HelmChartsDir, 0755); err != nil {
 		return err
 	}
+
+	projectName := strings.ToLower(s.config.ProjectName)
+
 	return machinery.NewScaffold().Execute(
 		s.newUniverse(),
 		&templates.GitIgnore{},
 		&templates.AuthProxyRole{},
 		&templates.AuthProxyRoleBinding{},
-		&metricsauth.AuthProxyPatch{},
+		&metricsauth.AuthProxyPatch{
+			OperatorName: projectName,
+		},
```

Comment:
Since we're using this in several places and it seems generally useful, wdyt about adding a `ProjectNameMixin` upstream and pulling that it so that our templates can make use of it?

Reply:
Great idea.

Reply:
https://github.com/kubernetes-sigs/kubebuilder/pull/1609